### ISSUE TITLE
SmartCity-Accessibility #283 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#179]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 134 lines...]
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: Jan 15, 2017
JxMaps license valid.
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 3.216 sec - in smartcity.accessibility.search.SearchTest
Running smartcity.accessibility.search.NearbyPlacesTest
doing the nearby search
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.198 sec <<< FAILURE! - in smartcity.accessibility.search.NearbyPlacesTest
nearByPlacesTest(smartcity.accessibility.search.NearbyPlacesTest)  Time elapsed: 0.197 sec  <<< ERROR!
com.teamdev.jxmaps.internal.aq: Uncaught Error: Missing parameter. You must specify location.
	at com.teamdev.jxmaps.internal.ao.a(SourceFile:60)
	at com.teamdev.jxmaps.PlacesService.nearbySearch(SourceFile:68)
	at smartcity.accessibility.search.NearbyPlacesSearch.findNearbyPlaces(NearbyPlacesSearch.java:40)
	at smartcity.accessibility.search.NearbyPlacesTest.nearByPlacesTest(NearbyPlacesTest.java:35)

Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 310.972 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Tests in error: 
  NearbyPlacesTest.nearByPlacesTest:35 » aq Uncaught Error: Missing parameter. Y...

Tests run: 45, Failures: 0, Errors: 1, Skipped: 4

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 05:48 min
[INFO] Finished at: 2017-01-15T18:13:11+00:00
[INFO] Final Memory: 28M/199M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] 36275db81b48f437cb1adf6cc74a55b768d37b13 - change in on right click, closes #178

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/283/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 134 lines...]
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: Jan 15, 2017
JxMaps license valid.
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 3.007 sec - in smartcity.accessibility.search.SearchTest
Running smartcity.accessibility.search.NearbyPlacesTest
doing the nearby search
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.162 sec <<< FAILURE! - in smartcity.accessibility.search.NearbyPlacesTest
nearByPlacesTest(smartcity.accessibility.search.NearbyPlacesTest)  Time elapsed: 0.16 sec  <<< ERROR!
com.teamdev.jxmaps.internal.aq: Uncaught Error: Missing parameter. You must specify location.
	at com.teamdev.jxmaps.internal.ao.a(SourceFile:60)
	at com.teamdev.jxmaps.PlacesService.nearbySearch(SourceFile:68)
	at smartcity.accessibility.search.NearbyPlacesSearch.findNearbyPlaces(NearbyPlacesSearch.java:40)
	at smartcity.accessibility.search.NearbyPlacesTest.nearByPlacesTest(NearbyPlacesTest.java:35)

Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 311.933 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Tests in error: 
  NearbyPlacesTest.nearByPlacesTest:35 » aq Uncaught Error: Missing parameter. Y...

Tests run: 45, Failures: 0, Errors: 1, Skipped: 4

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 05:50 min
[INFO] Finished at: 2017-01-15T18:26:48+00:00
[INFO] Final Memory: 29M/200M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] 36275db81b48f437cb1adf6cc74a55b768d37b13 - change in on right click, closes #178



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/284/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
